lib/testing/src/lib/core/pages/data-table-component.page.ts

```diff
@@ -166,6 +166,13 @@ export class DataTableComponentPage {
         BrowserVisibility.waitUntilElementIsVisible(element(by.id('adf-context-menu-content')));
     }
 
+    rightClickOnRowByIndex(index: number) {
+        BrowserActions.closeMenuAndDialogs();
```

Comment:
this should be handled in the actual tests, not here.
there could be scenarios where we would need to right click on an item and then right click on another (without closing the context menu)